data-raw/admiral_verify_templates.R

```diff
@@ -0,0 +1,238 @@
+# This script:  data-raw/admiral_verify_templates.R
+
+#
+# Generates ADaM from templates and compares to previously generated ADaM file (found in pharmaverseadam).
+# pharmaverseadam is the SOURCE.
+#
+# Much code taken from pharamavreseadam::create_adams_data.R
+# (https://github.com/pharmaverse/pharmaverseadam/blob/main/data-raw/create_adams_data.R)
+#
+# USAGE:   verify_templates()
+#
+# TODO:
+# - add `attr` to generated ADaM
+# - re-write `load_rda()`:  now serves two purposes and was written in simpler times
+# - where code overlaps, make pharamversadam script and this one the same.
+
+# Assumptions/Questions:
+# - ignore *.rda files in admiral/data (per Ben)
+# - written as standalone R script, not as package R function
+# - compares full ADaM - all rows (ie no reduction in number of rows in each dataset)
+# - use cli:: for messages/errors?
+
+#' (if were to add to `admiral` package)
```

Comment:
discuss

Reply:
if you want to, but I'm okay to keep it simple.  we are the only ones reading these messages not users